code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -51,9 +51,9 @@
 	var/update_flags = STATUS_UPDATE_NONE
 	if(prob(10))
 		to_chat(M, "<span class='danger'>Your insides are burning!</span>")
-		update_flags |= M.adjustToxLoss(rand(20,60)*REAGENTS_EFFECT_MULTIPLIER, FALSE)
+		update_flags |= M.adjustToxLoss(rand(2,6)*REAGENTS_EFFECT_MULTIPLIER, FALSE) // avg 0.4 toxin per cycle, not unreasonable
```

Comment:
`REAGENTS_EFFECT_MULTIPLIER` is equal to `0.4 / 0.4`, so... 1. God knows why this is even here. But anyway, this means that they'll take 2-6 toxin damage per cycle.

Reply:
Do you ever see toxins used anyways? I think slime blood would be a good use for it myself